go.mod

```diff
@@ -7,30 +7,29 @@ require (
 	github.com/csi-addons/volume-replication-operator v0.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/logr v0.4.0
-	github.com/onsi/ginkgo v1.16.1
-	github.com/onsi/gomega v1.11.0
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.14.0
 	github.com/open-cluster-management/api v0.0.0-20210527013639-a6845f2ebcb1
-	github.com/open-cluster-management/multicloud-operators-placementrule v1.0.1-2020-06-08-14-28-27.0.20201118195339-05a8c4c89c12
-	github.com/open-cluster-management/multicloud-operators-subscription v1.2.2-2-20201130-59f96
+	github.com/open-cluster-management/multicloud-operators-foundation v0.0.0-20210722145122-534b4232f1c9
+	github.com/open-cluster-management/multicloud-operators-placementrule v1.2.2-2-20201130-98cfd.0.20210722134723-73b5f55a3813
+	github.com/open-cluster-management/multicloud-operators-subscription v1.2.3-1-20210407-9f32325.0.20210722135022-ea2db38695ef
 	github.com/pkg/errors v0.9.1
-	go.uber.org/zap v1.15.0
-	k8s.io/api v0.20.5
-	k8s.io/apimachinery v0.21.2
+	go.uber.org/zap v1.18.1
+	k8s.io/api v0.22.0
+	k8s.io/apimachinery v0.22.0
 	k8s.io/client-go v12.0.0+incompatible
-	sigs.k8s.io/controller-runtime v0.8.3
+	sigs.k8s.io/controller-runtime v0.9.5
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.20.5
-
-// below: temporary measure to include multicloud-operators-foundation project with ManagedClusterView through forked repo
-require github.com/tjanssen3/multicloud-operators-foundation/v2 v2.0.0-20210512222428-660109db7f1d
-
 replace (
+	github.com/deislabs/oras => github.com/deislabs/oras v0.11.1
```

Comment:
oras v0.10.x was the original dependency, but that has a weird docker replace directive, which is corrected in the v.11.1 go.mod file.